### DISCUSSION TITLE
Github 정리 내용 및 사용방법 설명

### DISCUSSION THREAD

**Discussion by uush** [donghyun819/AutoCurtain_DY5th#29] [Announcements]
## 📢 Discussion 기능 활성화

- 작업 진행 중에 생긴 **의논사항, 질문** 등을 적어 주시면 댓글 형태로 내용을 주고 받을 수 있습니다.
- 또한, 문제사항에 대한 **도움요청, 새로운 아이디어 공유**도 이곳에 적어 주세요.
- issue에 적지 않은 디테일한 사항이나 보고 내용을 적으셔도 됩니다.

## 📝 Issues는?
- 이슈는 **작업 전**, 앞으로 각자 해야 할 작업 내용을 간단하게 요약한 내용을 적습니다. 
- 체크리스트로 작업 진행 사항을 알 수 있도록 해 주세요. **작업한 후**에는 꼭 체크해 주세요! 
- **작업 후**에 알려야 할 사항은 comment로 적어주세요.
- **Labels**와 **Assignees**를 지정해 주세요.
- 이슈에는 템플릿을 적어 놓았습니다. 새로운 이슈를 작성하실 때, **New task**를 참고해 주세요! 
(템플릿에 적힌 내용은 삭제한 후에 그 형식에 맞게 작성하시면 됩니다.)
- 그 외 알려야 할 사항은 Discussion에 적습니다.

## 📌Pull Request!
- 개인 branch에서 자신이 맡은 파트 작업이 완료되었다고 생각 되면 main에 반영하도록 요청하는 작업입니다.
- **main**은 완성된 코드들의 집합이므로 커튼 개폐기 작동을 위해 사용될 최종 코드입니다.
- 따라서 작업이 완료되면 Pull request를 통해 **요청 > 관리자 승인 > merge**되는 순서지만 저희는 회의에서 논의한 후에 merge하도록 합시다!
- Pull request가 승인 되면 개인 파트 작업이 완료된 것으로 생각하고 merge한 후에 개인 branch를 삭제하면 됩니다.
- 승인을 하려면 작업 내용을 이해할 수 있어야 하니 **주석**이나 **기능 설명**을 자세히 적어주시면 감사하겠습니다.
- Pull request에서 코드를 한줄한줄 살펴보면서 피드백할 수 있습니다. 
- **Reviewer와 Link issue**를 지정해 주세요. 컴토자와 관련 이슈 링크를 가져와 이해하기 쉽도록 하기 위함입니다.
- 합병하는 과정에서 문제가 발생할 수 있습니다. 이 문제는 개인이 해결할 수 있도록 합니다. 
> 제가 Pull request에 참고했던 링크를 공유 합니다. https://brunch.co.kr/@anonymdevoo/9

## 📊Milestone
- 설정한 기간동안 수행할 lssues을 등록해 놓으면 Open된 issue와 Close된 issue를 카운트하여 Percentage를 보여줍니다.
- Issues > Milestones 에서 New milestone을 만드시면 됩니다.
- Due Date 에 Milestone 종료 일자를 설정해 줍니다. 자동으로 Milestone이 종료되며 퍼센트가 계산됩니다.
- Description 에 Milestone의 상세한 설명을 적습니다.
- issue를 작성할 때 Milestone을 지정해 주셔도 되고, 자신이 만든 Milestone내에서 issue를 생성하셔도 됩니다.
- Milestone의 Due data가 종료되면 반드시 Close해 주세요.

그 외 기능에 대해서는 공부 후에 다시 적도록 하겠습니다. ( •̀ ω •́ )✧

**Comment by uush** [comment]
## 📢Project 기능을 활성화 하였습니다! 
이슈를 템플릿대로 적어주시면 **To do, In progress, Done**으로 나눠서 진행상황을 눈으로 볼 수 있도록! 이슈를 찾기 쉽도록! 나누겠습니다.

**Comment by IamJunhee** [comment]
기존 Issue에 있던 Discussion들은 모두 Discussion으로 이동해주세요!!

제가 직접 하려고 했는데, 권한이 없어서 못하네요

@uush
